packages/admin-ui-react/src/system-deprecated/tag.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react'
 import { isFunction, capitalize } from '@vtex/admin-ui-util'
 
-import { useSystem } from './context'
+import { useSystem } from '../system'
```

Comment:
tag is a deprecated feature or only its current implementation?

Reply:
the current implementation